src/pages/configeditor/migrating_data.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Configuration Editor Migration"
+excerpt: "Configuration Editor Migration."
+categories: configeditor
+slug: migration
+toc: true
+---
+
+## Migrating Early Access Program Data
+
+Customers that were part of the Early Access Program (EAP) can follow these steps to migrate existing artifacts to the newly installed Merative ACD Configuration Editor instace.
+
+1.  Stop the Merative Configuration Editor from the `acd-ce` diretory.
+
+    ```
+    ./run-acd-ce.sh -stop
+    ```
+
+1.  Navigate to the %HOME%/installs/config/artrifactstore directory in the EAP enviroment.
```

Comment:
typo:  should be "artifactstore"
typo:  should be "environment"
Question:  Can we clarify it as the IBM Watson ACD Configuration Editor EAP environment here or in the header "Migrating Early Access Data"?